runtime/opt/taupage/init.d/02-prepare-td-agent.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Ensure that directories are writable
+chmod -R 1777 /tmp/
```

Comment:
Again, why?

Reply:
Same as above https://github.com/zalando-stups/taupage/pull/511#discussion_r220101407.

Reply:
This doesn't appear to be necessary anymore.